portal-frontend/package.json

```diff
@@ -19,11 +19,13 @@
     "@angular/core": "^15.1.1",
     "@angular/forms": "^15.1.1",
     "@angular/material": "^14.0.5",
+    "@angular/material-moment-adapter": "^14.2.7",
```

Comment:
"@angular/material-moment-adapter": "^15.1.1" since we upgraded to Angular 15.

Reply:
done